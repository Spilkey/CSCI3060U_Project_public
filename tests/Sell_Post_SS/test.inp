loginuser04sellairpods meetup100.0026buyairpods meetup4user04logout
exit