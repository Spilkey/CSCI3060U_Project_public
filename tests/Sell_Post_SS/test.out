03 airpods meetup            user04        026 100.0004 airpods meetup            user04        010 100.0000 user04          SS 000000.00